### PR TITLE
fix how genus determined

### PR DESCRIPTION
**Pull request by ehwenk**
For genera that are hybrids the genus name is "x genus" - this was causing problems in AusTraits, because I'd only included an ifelse to hunt for "x" in some places

### CODE REVIEW

File: R/update_taxonomy.R [hunk 1]
```diff
@@ -369,7 +369,7 @@ update_taxonomy_APC_family <- function(data, resources) {
 # Function to update names of taxa whose aligned_names are
 # taxon_rank = species/infraspecific and taxonomic_dataset = APC
 update_taxonomy_APC_species_and_infraspecific_taxa <- function(data, resources, taxonomic_splits) {
-
+#browser()
```
Comment (dfalster) [inline R/update_taxonomy.R:372]: remove this!

File: R/standardise_names.R [hunk 2]
```diff
@@ -98,3 +98,27 @@ standardise_names <- function(taxon_names) {
...
+#' @return The genus for a scientific name.
+#'
+#' @examples
+#' genus = extract_genus(stripped_name)
+#' 
+#' 
+extract_genus <- function(taxon_name) {
+  genus <- 
+    ifelse(
+      stringr::word(taxon_name, 1) == "x",
```
Comment (dfalster) [inline R/standardise_names.R:119]: wouldn't the x be the second name? i.e. "Aaa x Bbb"

**Review by dfalster — DISMISSED** [review]
Approved once changes addressed

File: R/standardise_names.R (hunk 2 above)
Comment (ehwenk) [reply R/standardise_names.R:119]: No this is about genera that are "x AAA" - so hybrid genera, not hybrid taxa

**Review by dfalster — APPROVED** [review]

**Pull request merged**